File: VB/WinForms_Localizer/WinForms_Localizer/DesignerForm1.vb
Imports DevExpress.DashboardCommon.Localization
Imports DevExpress.DashboardWin.Localization
Imports DevExpress.XtraBars

Namespace WinForms_Localizer

    Public Partial Class DesignerForm1
        Inherits Ribbon.RibbonForm

        Public Sub New()
            DashboardWinLocalizer.Active = New MyDashboardWinLocalizer()
            DashboardLocalizer.Active = New MyDashboardCoreLocalizer()
            InitializeComponent()
            dashboardDesigner.LoadDashboard("..\..\Dashboards\dashboard1.xml")
        End Sub
    End Class
End Namespace

File: CS/WinForms_Localizer/WinForms_Localizer/DesignerForm1.cs
using DevExpress.DashboardCommon.Localization;
using DevExpress.DashboardWin.Localization;
using DevExpress.XtraBars;
using DevExpress.XtraMap;
using System;
using System.Data;
using System.Windows.Forms;

namespace WinForms_Localizer
{
    public partial class DesignerForm1 : DevExpress.XtraBars.Ribbon.RibbonForm
    {
        public DesignerForm1()
        {
            DashboardWinLocalizer.Active = new MyDashboardWinLocalizer();
            DashboardLocalizer.Active = new MyDashboardCoreLocalizer();
            InitializeComponent();
            dashboardDesigner.LoadDashboard(@"..\..\Dashboards\dashboard1.xml");


        }
    }
}
